src/main/java/fr/minuskube/inv/InvListener.java

```diff
@@ -0,0 +1,159 @@
+/*
```

Comment:
I would move this into a separate package like fr.minuskube.inv.internal to prevent clutter in main package